repos/system_upgrade/el7toel8/actors/peseventsscanner/libraries/peseventsscanner.py

```diff
@@ -475,44 +486,55 @@ def map_repositories(packages):
 
     if repo_without_mapping:
         report_skipped_packages('packages will not be installed or upgraded due to repositories unknown to leapp:',
-                                repo_without_mapping)
+                                repo_without_mapping,
+                                "Please file a bug in http://bugzilla.redhat.com/ for leapp-repository component of"
+                                "the Red Hat Enterprise Linux 7 product.")
 
 
-def report_skipped_packages(message, packages):
+def report_skipped_packages(message, packages, remediation=None):
     """Generate report message about skipped packages"""
     packages = sorted(packages)
     title = 'Packages will not be installed'
```

Comment:
Unfortunately, this title is duplicate - it's used for two report messages:
```
Risk Factor: high
Title: Packages will not be installed
Summary: 4 packages will not be installed or upgraded due to repositories unknown to leapp:
- libselinux-python3
- postfix-cdb
- postfix-pcre
- postfix-sqlite
Remediation: [hint] Please file a bug in http://bugzilla.redhat.com/ for leapp-repository component ofthe Red Hat Enterprise Linux 7 product.
----------------------------------------
Risk Factor: high
Title: Packages will not be installed
Summary: 2 packages will be skipped because they are available only in RHEL 8 repositories that are intentionally excluded from the list of repositories used during the upgrade. See the report message titled "Excluded RHEL 8 repositories"
 for details.
 The list of these packages (repositories: codeready-builder-for-rhel-8-x86_64-rpms):
- python3-pyxattr
- rpcgen
```
I'll create a quick PR to make them different. @AloisMahdal, is it going to break your tests?

Reply:
Ok, I've talked to @ZhukovGreen and he'll change the message titles.
I suggest:
- `Packages from unknown repositories will not be installed`
- `Packages available in excluded repositories will not be installed`